services/api-server/src/simcore_service_api_server/api/routes/studies_jobs.py

```diff
@@ -302,15 +302,24 @@ async def get_study_job_outputs(
     return job_outputs
 
 
-@router.post(
-    "/{study_id}/jobs/{job_id}/outputs/logfile",
-    response_class=RedirectResponse,
-    include_in_schema=API_SERVER_DEV_FEATURES_ENABLED,
-    status_code=status.HTTP_501_NOT_IMPLEMENTED,
+@router.get(
+    "/{study_id}/jobs/{job_id}/outputs/log-links",
+    response_model=LogLinkMap,
+    status_code=status.HTTP_200_OK,
+    summary="Get download links for study job log files",
 )
-async def get_study_job_output_logfile(study_id: StudyID, job_id: JobID):
-    msg = f"get study job output logfile study_id={study_id!r} job_id={job_id!r}. SEE https://github.com/ITISFoundation/osparc-simcore/issues/4177"
-    raise NotImplementedError(msg)
+async def get_study_job_output_logfile(
+    study_id: StudyID,
+    job_id: JobID,
+    user_id: Annotated[PositiveInt, Depends(get_current_user_id)],
+    director2_api: Annotated[DirectorV2Api, Depends(get_api_client(DirectorV2Api))],
+):
+    msg = f"get study job output logfile study_id={study_id!r} job_id={job_id!r}."
```

Comment:
TIP: consider using `servicelib.logging_utils.log_decorator` on the handler function or  `servicelib.logging_utils.log_context` in the plementation

Reply:
Done